docs/API.md

```diff
@@ -104,132 +105,157 @@ gulp.task('somename', function() {
 
-- give it a hint to tell it when the task is done,
-- and give it a hint that a task depends on completion of another.
+gulp.task('clean', function() {
+  return promisedDel(['.build/']);
+});
+```
 
-For these examples, let's presume you have two tasks, "one" and "two" that you specifically want to run in this order:
+##### Return a child process
 
-1. In task "one" you add a hint to tell it when the task is done.  Either take in a callback and call it when you're
-done or return a promise or stream that the engine should wait to resolve or end respectively.
+```js
+gulp.task('clean', function() {
+  return spawn('rm', ['-rf', path.join(__dirname, 'build')]);
+});
 
-2. In task "two" you add a hint telling the engine that it depends on completion of the first task.
+```
 
-So this example would look like this:
 
-```js
-var gulp = require('gulp');
+### gulp.parallel(tasks, ...)
+
+Takes a variable amount of task names or function and returns
+a function of the composed tasks or functions. 
+
+When the returned function is executed, the tasks or functions will be executed in parallel, 
+all being executed at the same time. If an error occurs, all execution will complete.
 
-// takes in a callback so the engine knows when it'll be done
-gulp.task('one', function(cb) {
-    // do stuff -- async or otherwise
-    cb(err); // if err is not null and not undefined, the run will stop, and note that it failed
+```js
+gulp.task('one', function(done) {
+  // do stuff
+  done();
 });
 
-// identifies a dependent task must be complete before this one begins
-gulp.task('two', ['one'], function() {
-    // task 'one' is done now
+gulp.task('two', function(done) {
+  // do stuff
+  done();
 });
 
-gulp.task('default', ['one', 'two']);
+gulp.task('default', gulp.parallel(['one', 'two', function(done) {
+  // do more stuff
+  done();
+}]));
 ```
 
+#### tasks
+Type: `Array`, `String` or `Function`
 
-### gulp.watch(glob [, opts], tasks) or gulp.watch(glob [, opts, cb])
+A task name, a function or an array of either.
 
-Watch files and do something when a file changes. This always returns an EventEmitter that emits `change` events.
 
-### gulp.watch(glob[, opts], tasks)
+### gulp.series(tasks, ...)
 
-#### glob
-Type: `String` or `Array`
+Takes a variable amount of task names or function and returns a function 
+of the composed tasks or functions.
 
-A single glob or array of globs that indicate which files to watch for changes.
+When the returned function is executed, the tasks or functions will be executed 
+in series, each waiting for the prior to finish. If an error occurs, 
+execution will stop.
 
-#### opts
-Type: `Object`
+```js
+gulp.task('one', function(done) {
+  // do stuff
+  done();
+});
 
-Options, that are passed to [`gaze`](https://github.com/shama/gaze).
+gulp.task('two', function(done) {
+  // do stuff
+  done();
+});
+
+gulp.task('default', gulp.series(['one', 'two', function(done) {
+  // do more stuff
+  done();
+}]));
+```
 
 #### tasks
```

Comment:
@contra I believe we discussed this in the past and decided it wasn't worth breaking the syntax on watch and the array would just execute in parallel.  I remember rebasing to add the functionality back in after I removed.  I'd be fine with removing the array syntax if we want to change the decision (it simplifies .watch).